packages/files-ui/src/Contexts/UserContext.tsx

```diff
@@ -14,16 +14,19 @@ export type Profile = {
   publicAddress?: string
   email?: string
   createdAt?: Date
+  username?: string
 }
 
 interface IUserContext {
   profile: Profile | undefined
   refreshProfile(): Promise<void>
-  updateProfile(
+  updateProfile: (
     firstName: string,
     lastName: string,
     // email: string,
-  ): Promise<void>
+  ) => Promise<void>
+  lookupOnUsername: (username: string) => Promise<boolean | undefined>
```

Comment:
undefined not needed (see next comment)
```suggestion
  lookupOnUsername: (username: string) => Promise<boolean>
```